File: csharp/nuget/src/Vosk.cs
namespace Vosk {

public class Vosk {
  public static void SetLogLevel(int level) {
    VoskPINVOKE.SetLogLevel(level);
  }

  public static void GpuInit() {
    VoskPINVOKE.GpuInit();
  }

  public static void GpuThreadInit() {
    VoskPINVOKE.GpuThreadInit();
  }
}

}

File: kotlin/src/jvmMain/kotlin/org/vosk/Vosk.kt
package org.vosk


/**
 * 26 / 12 / 2022
 */
actual object Vosk {
	actual fun setLogLevel(logLevel: LogLevel) {
		LibVosk.vosk_set_log_level(logLevel.value)
	}

	actual fun gpuInit() {
		LibVosk.vosk_gpu_init()
	}

	actual fun gpuThreadInit() {
		LibVosk.vosk_gpu_thread_init()
	}
}